reana_commons/config.py

```diff
@@ -336,6 +336,11 @@ def kubernetes_node_label_to_dict(node_label):
 SHARED_VOLUME_PATH = os.getenv("SHARED_VOLUME_PATH", "/var/reana")
 """Default shared volume path."""
 
+DEFAULT_WORKSPACE_PATH = os.getenv("DEFAULT_WORKSPACE_PATH", "/var/reana")
+"""Default workspace path defined by the admin."""
+
+ADMIN_ALLOWED_WORKSPACES = os.getenv("ADMIN_ALLOWED_WORKSPACES", None)
```

Comment:
Also, please see [my comment](https://github.com/reanahub/reana/pull/533/files#r700897170) about the name